{{ cookiecutter.folder_name }}/.travis.yml

```diff
@@ -9,36 +14,75 @@ env:
       - secure: ""
       {% endif %}
 
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/.ccache  # https://github.com/travis-ci/travis-ci/issues/5853
 
-python:
-  # We don't actually use the Travis Python, but this keeps it organized.
-  - "3.6"
+matrix:
+  include:
+    - python: 3.6
```

Comment:
I think one of these environment setups needs a special variable to designate when to build/push the documentation. Right now it looks like it woll push on both `pcds-tag` builds.

Reply:
Nice catch @n-wbrown!

Reply:
You're right. Looks like it could go in the pcds-tag 3.6 one, just like `CONDA_UPLOAD`?

Reply:
Do we need `CONDA_UPLOAD` at all?

Reply:
I think you're right. I copied these in and modified them a bit from other projects. I think they'll need some reviewing, too, at some point.